hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/operations/AbstractJoinOperation.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.cluster.impl.operations;
+
+import com.hazelcast.internal.cluster.impl.ClusterDataSerializerHook;
+import com.hazelcast.spi.Operation;
+
+public abstract class AbstractJoinOperation extends Operation implements JoinOperation {
```

Comment:
Good point, maybe for readibility let's keep it? I mean if there's more maybe someone will forget to override the returnsResponse. But thx for the comment